### ISSUE TITLE
fribidi-types.h: Fatal error:  'config.h' file not found

### ISSUE THREAD

**Issue by ihaveadrame** [python-pillow/Pillow#3180]
### What did you do?
build libraqm on macOS
option:
$ brew install libtiff libjpeg webp little-cms2
$ brew install freetype harfbuzz fribidi
$ depends/install_raqm_cmake.sh

### What did you expect to happen?
Building on macOS

### What actually happened?

-- Build files have been written to: /Users/alan/fork_work/Pillow/depends/raqm-cmake-b517ba80/build
Scanning dependencies of target raqm
[ 50%] Building C object CMakeFiles/raqm.dir/src/raqm.c.o
In file included from /Users/alan/fork_work/Pillow/depends/raqm-cmake-b517ba80/src/raqm.c:31:
In file included from /usr/local/include/fribidi/fribidi.h:31:
In file included from /usr/local/include/fribidi/fribidi-unicode.h:32:
/usr/local/include/fribidi/fribidi-types.h:31:11: fatal error: 'config.h' file not found
\# include <config.h>
          ^~~~~~~~~~
1 error generated.
make[2]: *** [CMakeFiles/raqm.dir/src/raqm.c.o] Error 1
make[1]: *** [CMakeFiles/raqm.dir/all] Error 2
make: *** [all] Error 2
~/fork_work/Pillow/depends
/usr/local/include/fribidi/fribidi-types.h:31:11: fatal error: 'config.h' file not found

### What versions of Pillow and Python are you using?
python2.7.14
Pillow5.1.0

**Comment by radarhere** [comment]
What version of MacOS are you using?

**Comment by ihaveadrame** [comment]
@radarhere macOS 10.13.3

**Comment by aclark4life** [comment]
@ihaveadrame I can reproduce this, but closing since this is fribidi failing to build, not Pillow (and while we apparently offer these scripts to help build raqm, we can only take responsibility for so much …) Please reopen if you disagree.

**Comment by anpr** [comment]
I have the exact same error. Can somebody report if they've been able to fix this?

**Comment by aclark4life** [comment]
@anpr No, still reproducible for me:

```

-- Build files have been written to: /Users/alexclark/Developer/pillow/Pillow/depends/raqm-cmake-b517ba80/build
Scanning dependencies of target raqm
[ 50%] Building C object CMakeFiles/raqm.dir/src/raqm.c.o
In file included from /Users/alexclark/Developer/pillow/Pillow/depends/raqm-cmake-b517ba80/src/raqm.c:31:
In file included from /usr/local/include/fribidi/fribidi.h:31:
In file included from /usr/local/include/fribidi/fribidi-unicode.h:32:
/usr/local/include/fribidi/fribidi-types.h:31:11: fatal error: 'config.h' file not found
# include <config.h>
          ^~~~~~~~~~
1 error generated.
make[2]: *** [CMakeFiles/raqm.dir/src/raqm.c.o] Error 1
make[1]: *** [CMakeFiles/raqm.dir/all] Error 2
make: *** [all] Error 2
~/Developer/pillow/Pillow/depends

```

**Comment by aclark4life** [comment]
On second thought, maybe we should fix this 😄 @python-pillow/pillow-team Anyone know what's going on here?

**Comment by radarhere** [comment]
Looks like HAVE_CONFIG_H is being defined by raqm, and then read by fribidi, which doesn't have a config.h. While this has been corrected by fribidi/fribidi#85 and fribidi/fribidi@b534ab2, a new version of fribidi hasn't been released since then. I've created https://github.com/HOST-Oman/libraqm-cmake/pull/1 as a workaround for raqm.

**Comment by radarhere** [comment]
The workaround was merged, and so I've created PR #3818 to resolve this.

**Comment by hugovk** [comment]
@dov Hello, is there a schedule for the next release of GNU FriBidi? Thank you!
